src/Tasks/SwitchSessionConfigTask.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Spatie\Multitenancy\Tasks;
+
+class SwitchSessionConfigTask implements SwitchTenantTask
+{
+    private $originalConnection;
+
+    private $originalDriver;
+
+    public function __construct()
+    {
+        $this->originalDriver = config('session.driver');
+        $this->originalConnection = config('session.connection');
```

Comment:
I think this should look like this: `$this->originalConnection = config('session.connection') ?? config('database.default');
`

Because the `session.connection` can be null by default

Reply:
Sorry, I don't see your point... Any problem if it is `null`?

Reply:
The `setSessionDatabaseConnection` function only accepts strings. It will fail on `null`